go.mod

```diff
@@ -82,3 +82,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+replace github.com/hashicorp/go-tfe => ../go-tfe
```

Comment:
Instead of this, you should update go.mod to reference your newest commit `98975f447700482a79e4d7d177cd406ff8132f1d` from go-tfe so we can see tests running and stuff

Reply:
That's helpful! I was waiting for the release but this makes sense